### DISCUSSION TITLE
Human Talks 🤵🏼

### DISCUSSION THREAD

**Discussion by Slashgear** [lyonjs/shortvid.io#118] [Conference / Meetup Communication]
Let's see what we can do to help "Human talks" meetup 👋🏼

**Comment by P0ppoff** [comment]
We need :
 * a video with 3-4 talks and speakers, the host info, the date
 * a video when a new talk are validated
 * a reminder one day and one week before event
 * a video for looking for speaker or for host
 * a video to thank the host, the day after

**Reply by P0ppoff** [reply]
All of the videos should be by city, same logo, but different city name

**Reply by Slashgear** [reply]
How can we proceed ? Do we work on dedicate template or generic ones ? 

Do you have any design idea ?

During February we will try with @CruuzAzul to make LyonJs template more generics and globally make this project usable by any meetup.

Do you want to contribute it yourself ?

**Reply by mickaelalvs** [reply]
If we want to work on generic templates, I don't know if Emeric (the trainee who will work on the project) will be able to start on this in February. But it might give us time to look at how we can do that. I'd like to see the LyonJS templates transition be a topic he deals with himself 😊
